File: tests/go/cases/case_203_closure_registry_delay_01.go
package escape_tests

var retainedCase203 = []map[string]string{}

func Case203ClosureRegistryDelay01(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "closure_registry_delay_01",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "closure_registry_delay_01:" + raw,
	}
	handler := func() string { retainedCase203 = append(retainedCase203, payload); return payload["input"] }
	_ = handler
	// ESCAPE: retained closure-like dispatch.
	retainedCase203 = append(retainedCase203, map[string]string{"h": payload["task"]})
	return "ok"
}

File: tests/python/cases/case_203_closure_registry_delay_01.py
"""
Task case 203: closure_registry_delay_01 deep stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_203_closure_registry_delay_01(input_data):
    task_name = 'closure_registry_delay_01'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'extreme',
        'stage': 'stress',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    def later() -> str:
        RETAINED_AUDIT.append(payload)
        return payload['input']
    # ESCAPE: retained closure mutates global sink when invoked later.
    RETAINED_HANDLERS.append(later)
    return 'ok'
